processor/tailsamplingprocessor/metrics.go

```diff
@@ -30,7 +30,7 @@ var (
 	statCountTracesSampled       = stats.Int64("count_traces_sampled", "Count of traces that were sampled or not per sampling policy", stats.UnitDimensionless)
 	statCountGlobalTracesSampled = stats.Int64("global_count_traces_sampled", "Global count of traces that were sampled or not by at least one policy", stats.UnitDimensionless)
 
-	statDroppedTooEarlyCount    = stats.Int64("sampling_trace_dropped_too_early", "Count of traces that needed to be dropped the configured wait time", stats.UnitDimensionless)
+	statDroppedTooEarlyCount    = stats.Int64("sampling_trace_dropped_too_early", "Count of traces that needed to be dropped before the configured wait time", stats.UnitDimensionless)
```

Comment:
Would it make sense to give more context in the description as to possible reasons why this error is hit? Or should we simply rely on the README to point users in the right direction?

Reply:
I've added an FAQ section to the README, it should be enough at this point.